src/adapters/hermez-api.ts

```diff
@@ -482,6 +440,48 @@ function createHermezAccount(
   };
 }
 
+function getHermezRawAccounts(
```

Comment:
Instead of having this `getHermezRawAccounts` function I would keep it simple and just keep the the `getHermezAccounts`. I don't like to introduce too much terminology like the `getXXRaw`, we already have the `unknownXXXParser` and we are going to get lost with all this stuff soon...

Reply:
Makes total sense. Done!